keras/callbacks.py

```diff
@@ -1782,7 +1782,8 @@ def __init__(self,
     elif mode == 'max':
       self.monitor_op = np.greater
     else:
-      if 'acc' in self.monitor:
+      if (self.monitor.endswith('acc') or self.monitor.endswith('accuracy') or
+        self.monitor.endswith('auc')):
```

Comment:
Fix indent

Reply:
I don't know fix what . The indent is two in ```keras``` code ...

Reply:
Add two spaces.

Reply:
Done.